### ISSUE TITLE
BUG: `opts.stopEarly` should not swallow `--`

### ISSUE THREAD

**Issue by micalevisk** [meszaros-lajos-gyorgy/minimist-lite#7]
Original issue by @julien-f at https://github.com/substack/minimist/issues/71

```js
console.log(require('minimist')(
  process.argv.slice(2),
  { stopEarly: true }
)._)
```

Expected result (everything after first non option):

```
> node index.js foo -- bar baz
[ 'foo', '--', 'bar', 'baz' ]
```

Actual result:

```
> node index.js foo -- bar baz
[ 'foo', 'bar', 'baz' ]
```

**Comment by micalevisk** [comment]
looks like this was fixed already in this open PR: https://github.com/substack/minimist/pull/77

**Comment by meszaros-lajos-gyorgy** [comment]
Hi! Sorry for the delay in response!

I'll add the fix after merging pull request #9.

**Comment by meszaros-lajos-gyorgy** [comment]
Released it as part of https://github.com/meszaros-lajos-gyorgy/minimist-lite/releases/tag/v2.1.0
